CHANGELOG.rst

```diff
@@ -781,7 +781,7 @@ Improved Documentation
   location ({envtmpdir}/pseudo-home).  If an index url was specified
   a .pydistutils.cfg file will be written with an index_url setting
   so that packages defining ``setup_requires`` dependencies will not
-  silently use your HOME-directory settings or https://pypi.python.org/pypi.
+  silently use your HOME-directory settings or PyPi.
```

Comment:
removed a link here?

Reply:
broken.

Reply:
my point was maybe we should have replaced it with http://pypi.org instead of removing it.